features/step_definitions/book_index_page_steps.rb

```diff
@@ -28,3 +29,16 @@
 Then(/^I am shown a list of books with that title$/) do
   expect(page).to have_content(Book.find(1).title)
 end
+
+Given(/^some books have been ordered$/) do
+  @books = Book.order(published_date: :desc)
```

Comment:
I think you mis-understand "ordered" here. I think it means purchased. You're doing that above in the `there are 100 books in the database`, but I think it should go in here.